cluster/images/nginx/README.md

```diff
@@ -0,0 +1,8 @@
+### Nginx image
+This image is used to run nginx on the master.
+
+#### Instructions
+docker build -t gcr.io/google-containers/nginx:<version>
```

Comment:
I am curious why we didnt want to use the official nginx docker container https://registry.hub.docker.com/_/nginx/tags/